pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -641,14 +641,15 @@ func (exp *FlowExporter) findFlowType(conn flowexporter.Connection) uint8 {
 }
 
 func (exp *FlowExporter) fillEgressInfo(conn *flowexporter.Connection) {
-	egressName, egressIP, err := exp.egressQuerier.GetEgress(conn.SourcePodNamespace, conn.SourcePodName)
+	egressName, egressIP, egressNode, err := exp.egressQuerier.GetEgress(conn.SourcePodNamespace, conn.SourcePodName)
```

Comment:
To add a new field, please open a PR in [go-ipfix](https://github.com/vmware/go-ipfix) repository to update the registry first.

reference PRs for adding egressName and egressIP fields: https://github.com/vmware/go-ipfix/pull/308 https://github.com/antrea-io/antrea/pull/5088

Reply:
thanks.